### PR TITLE
bump activities

### PR DESCRIPTION
**Pull request by mdulat**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1114,7 +1114,7 @@
       }
     },
     "@brightspace-ui-labs/multi-select": {
-      "version": "github:BrightspaceUILabs/multi-select#bbd0f7b2e1e9d148b1a2c261709ad43091c3b08a",
+      "version": "github:BrightspaceUILabs/multi-select#512f27c576a57a654dcfee1d41ea94c8b9b86e2a",
```
Comment (mdulat) [inline package-lock.json:1117]: @jstefaniuk-d2l

File: package-lock.json [hunk 2]
```diff
@@ -4443,7 +4443,7 @@
       }
     },
     "d2l-consistent-evaluation": {
-      "version": "github:BrightspaceHypermediaComponents/consistent-evaluation#8802c71d0b967776b7e983774c06192359dda02c",
+      "version": "github:BrightspaceHypermediaComponents/consistent-evaluation#b6cfdc2f387ff0ca186db41f11ad6fac43c3404b",
```
Comment (mdulat) [inline package-lock.json:4446]: @jeroach

File: package-lock.json [hunk 3]
```diff
@@ -4817,7 +4818,7 @@
       }
     },
     "d2l-my-courses": {
-      "version": "github:Brightspace/d2l-my-courses-ui#512ebf553fa8a104dab35e8401b7fc34e25f862a",
+      "version": "github:Brightspace/d2l-my-courses-ui#903d903b35f9e596fa1879f6af1ca9ef66299fbf",
```
Comment (mdulat) [inline package-lock.json:4821]: @svanherk

File: package-lock.json [hunk 4]
```diff
@@ -4924,7 +4925,7 @@
       "dev": true
     },
     "d2l-organizations": {
-      "version": "github:BrightspaceHypermediaComponents/organizations#36e1d28fe83c6b94a93cd711286708bf217ca88f",
+      "version": "github:BrightspaceHypermediaComponents/organizations#5203c9669df1b17ef69c1c27bfc8b6ec68daa987",
```
Comment (mdulat) [inline package-lock.json:4928]: @dlockhart

File: package-lock.json (hunk 3 above)
Comment (svanherk) [reply package-lock.json:4821]: Thanks @mdulat!

**Pull request merged**